### ISSUE TITLE
Particle3D scale does not work in billboard mode.

### ISSUE THREAD

**Issue by DimaKiva** [godotengine/godot#76328]
### Godot version

Godot_v4.1-dev1

### System information

Ubuntu 22.04, RTX3060ti

### Issue description

Particle3D scale does not work in billboard mode.

### Steps to reproduce

Add 3D particles, enable billboard particle.

### Minimal reproduction project

[Particles test.zip](https://github.com/godotengine/godot/files/11299596/Particles.test.zip)

**Comment by clayjohn** [comment]
Do you have keep_scale checked or unchecked?

**Comment by DimaKiva** [comment]
> Do you have keep_scale checked or unchecked?

Wow!
It seems that this was not required before. Thank you.

**Comment by clayjohn** [comment]
I think you are right. You are not the first person to encounter this issue. 

For context, the option was introduced in https://github.com/godotengine/godot/pull/65353. Previously the scale was always included, but it was included in a way that broke when using non-uniform scales or rotations. 

Enabling ``keep_scale`` by default introduces an extra matrix multiplication, but it ensures correct behaviour

**Comment by QbieShay** [comment]
Yes, I think that would be the best default behaviour. Clay, what do you think about making an extension of StandardMaterial that extends the class and just puts better defaults for particles? (unshaded, transparent, keep_scale on, etc.). No special shader code, just different defaults?

**Comment by clayjohn** [comment]
For future reference, I have looked into enabling ``keep_scale`` by default for only particles billboard mode and it will be quite difficult given how material flags are implemented as ``keep_scale`` is shared between the particles billboard modes. At this point since 4.0 has been out already for several months, we may just be stuck with having it off by default.

**Comment by QbieShay** [comment]
Perhaps we can have the particle node then pre-configure a spatial material for its first draw pass?

**Comment by clayjohn** [comment]
> Perhaps we can have the particle node then pre-configure a spatial material for its first draw pass?

That might be possible, as long as we could figure out a nice UX for it. Personally, I prefer just extending SpatialMaterial and having a "ParticlesRenderMaterial" that has good defaults for particles

**Comment by QbieShay** [comment]
Let's do that with a separate proposal then. I'll put it in my TODO list.
